examples/cpy/axpb.py

```diff
@@ -25,7 +26,8 @@ def data(n, backend):
 
 def compare(m=20):
     N = 2**np.arange(1, 25)
-    backends = [['cython', False], ['cython', True], ['opencl', False]]
+    backends = [['cython', False], ['cython', True], ['opencl', False],
+            ['cuda', False]]
```

Comment:
Indentation.  Also perhaps we should do some conditional imports in case the respective backends are not available.